backend/src/main/java/com/chingu/ChinguBoard/model/User.java

```diff
@@ -1,13 +1,19 @@
 package com.chingu.ChinguBoard.model;
 
+import java.util.Collection;
+import java.util.List;
+
 import org.springframework.data.annotation.Id;
 import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
 
-@Document
-public class User { 
-    // variable ID
+@Document(collection = "user")
+public class User implements UserDetails { 
+    
     @Id
-    private Long id;
+    private String id;
```

Comment:
why was Long switched to String?

Reply:
mostly for simplicity. MongoDB auto generates an `ObjectId` for you which can be represented as a string. I believe long can be used to auto generate sequential IDs but it requires some implementation of a generator